.flake8

```diff
@@ -4,9 +4,9 @@ exclude =
     __pycache__,
     build,
     dist,
-    versioneer.py,
-    labpdfproc/_version.py,
-    doc/manual/source/conf.py
+    doc/source/conf.py,
+    */doc/source/conf.py
+builtins = cookiecutter
```

Comment:
Changed this, so we no longer exclude `source.py` as the issue with that is it could not find the built in `cookiecutter` from our jinja2 template.